src/hostedPages/activities/cloudinary/cloudinary.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { Meta, Story } from '@storybook/react/types-6-0'
+import { CloudinaryExtensionProps } from './types'
+import { ThemeProvider } from '../../../atoms'
+import { HostedPageLayout } from '../../layouts/HostedPageLayout/HostedPageLayout'
+import { CloudinaryUpload as CloudinaryUploadComponent } from './CloudinaryUpload'
+import image from '../../../assets/link.svg'
+
+export default {
+  title: 'HostedPages/Activities/Cloudinary/CloudinaryUpload',
+  component: CloudinaryUploadComponent,
+  argTypes: {
+    cloudName: {
+      control: 'text',
+      defaultValue: 'dip5c2e0n',
+    },
+    uploadPreset: {
+      control: 'text',
+      defaultValue: 'mufwljl2',
```

Comment:
[C] Make sure we are not committing actual/real upload presets to the repo. If we do so, people testing this will be able to upload files to that Cloudinary account.